packages/core/src/App/Containers/SetResidenceModal/set-residence-modal.jsx

```diff
@@ -76,6 +76,7 @@ const SetResidence = ({ enableApp, onSetResidence, residence_list, toggleModalVi
                                     is_disabled={!values.residence || !!errors.residence || isSubmitting}
                                     text={localize('Set residence')}
```

Comment:
can we pass this text as children and pass align as prop
<Text align='right'>  <Localize i18n_default_text='Set residence' /></Text>